config/index.js

```diff
@@ -790,6 +790,20 @@ var conf = convict({
     format: 'String',
     env: 'SENTRY_DSN'
   },
+  totp: {
+    serviceName: {
+      doc: 'Default service name to appear in authenticator',
+      default: 'Firefox',
+      format: 'String',
+      env: 'TOTP_SERVICE_NAME'
+    },
+    step: {
+      doc: 'Default time step size (seconds)',
+      default: 30,
+      format: Number,
```

Comment:
Should this be `'nat'` rather than `Number`? Presumably we'll never want negatives or floats...